### ISSUE TITLE
Free production with custom Production input

### ISSUE THREAD

**Issue by fpvandoorn** [SifVerT/SifDocs#5]
I made a game with x3 Production input and x2 Production output. 
I noticed that many constructors, when they have less than the required ingredients (but more than 1/3 the ingredients, i.e. more than the regular cost) they would continue production. And the constructor wouldn't consume the ingredients in the input, which means the constructor would run indefinitely, creating the output products out of thin air.

**Comment by Jack-L-22** [comment]
Where did you do the config? Docu doesn't mention the configuration method.

**Comment by fpvandoorn** [comment]
This is in "Difficulty Tuner" > "Recipe Production output and Ingredient" 

(there might be a better place to report bugs, but I didn't find it)

**Comment by SifVerT** [comment]
Thanks for the bug report. I'll keep the issue open.

**Comment by SifVerT** [comment]
It could be, hypothetically, that the recipe is already in use by the constructor and continue to use the old values for input, even if it links visually to the altered recipe. Might need to change away from the recipe for another and then select it again. It could be a cached value which the constructor uses internally in itself to save on the computer processing.

It doesn't help reloading the save game?

**Comment by fpvandoorn** [comment]
No, this definitely happened when setting difficulty tuner at the very start of the game (before building any constructors) and persisted/reappeared on reloads.

Note that "using the old values for the inputs" doesn't fully explain the behavior, since the input value was not subtracted from the input amount each production cycle.

**Comment by SifVerT** [comment]
Ok, so it appears the recipes show the altered values but the machines use another list.
